TWLight/applications/views.py

```diff
@@ -450,16 +452,17 @@ def dispatch(self, request, *args, **kwargs):
         if self._get_partners()[0].authorization_method == Partner.BUNDLE:
             raise PermissionDenied
         elif self._get_partners()[0].status == Partner.WAITLIST:
-            # Translators: When a user applies for a set of resources, they receive this message if none are currently available. They are instead placed on a 'waitlist' for later approval.
             messages.add_message(
                 request,
                 messages.WARNING,
-                _(
-                    "This partner "
+                # fmt: off
+                # Translators: When a user applies for a set of resources, they receive this message if none are currently available. They are instead placed on a 'waitlist' for later approval.
+                _("This partner "
                     "does not have any access grants available at this time. "
                     "You may still apply for access; your application will be "
                     "reviewed when access grants become available."
                 ),
```

Comment:
closing brackets for `_()` should be lifted to the same line as the source string